### ISSUE TITLE
p.giss.apoll.1 lin 7

### ISSUE THREAD

**Issue by danielrruf** [papyri/idp.data#44]
the second lb in 

`<lb n="7" break="no"/>πρόν. οὐκ ἀγνοεῖς δὲ ὅτι <app type="editorial"><lem resp="BL 12.149-150">ἡμιλί<lb n="8" break="no"/>τριον</lem><rdg>ἡμῖν`

most likely is `<lb n="7" break="no"/>`

**Comment by jcowey** [comment]
This is quite correct as far as I can see. Made some corrections to minor mistakes, but basically this is correctly marked up.

https://github.com/papyri/idp.data/commit/ed012d033eb1ef210791fcd6751bcdf697582261
